Makefile

```diff
@@ -133,6 +133,15 @@ else
 	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoLinkE2e" -ginkgo.succinct
 endif
 
+# Run link e2e tests
+.PHONY: test-watch-e2e
+test-watch-e2e:
+ifdef TIMEOUT
+	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoWatchE2e" -ginkgo.v -timeout $(TIMEOUT)
```

Comment:
I have it ginkgo.v for the moment for debugging
It'll be replaced with succinct once tests are green

Reply:
Yeah I think that would be nice

Reply:
I was not confident about the use of tag ```-ginkgo.v``` in test debugging. Now onward i will start using the same flag as my test debugging solution.

@anmolbabu Once it solves your purpose then do not forget to replace with ```-ginkgo.succinct``` before merge ;)